### PR TITLE
Remove prepareWebSymlinks

### PR DESCRIPTION
**Pull request by fkubis**

### CODE REVIEW

**Comment by marcj** [comment]
For the sake of history: We remove that function as communicated in gitter. Since prepareWebSymlinks forces the user to have symlinks, and that is not everywhere possible (vagrant) we removed that functionality so its up to the developer whether they want to populate web/bundles via symlinks or copy. Its actually also already implemented in the composer.json hooks after each composer update/install, so there's no need to do it again using JarvesBundle.

@fkubis Thanks for contributing!

**Comment by marcj** [comment]
Travis failed due the missing `install:assets` call in the `before_script` section.

**Comment by fkubis** [comment]
Fails agains -_ couse its called `assets:install` I try again..

**Comment by fkubis** [comment]
Any hint how to use the symfony console in travis?

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ env:
 before_script:
   - gem install sass
   - composer install --prefer-dist
+  - php bin/console assets:install
```
Comment (marcj) [inline .travis.yml]: should be `php Tests/Integration/skeletion/app/console assets:install` like in the lines below

**Comment by fkubis** [comment]
Anyway the symfony assets:install does not compile the sass files. 

Maybe we should use gulp with libsass, uglify and bower to handle the sass, js and css? I would prefer such thing. They are really common.

**Comment by marcj** [comment]
> Anyway the symfony assets:install does not compile the sass files.

correct. Thats not the intent of assets:install.

Its description:

> The assets:install command installs bundle assets into a given
 directory (e.g. the web directory).
>
>   php Tests/Integration/skeletion/app/console assets:install web
>
> A "bundles" directory will be created inside the target directory and the
 "Resources/public" directory of each bundle will be copied into it.

Using gulp/grunt makes only sense for websites where you explicitly want to have a custom build for frontend assets. For generic themes that would make no sense as it would force a user that wants to install a new theme to run some kind of gulp script. However, if someone has a bigger project they are free to use whatever frontend build tools they want. So, Jarves does not force anyone to use internal sass compiler. It's there for rapid prototyping and a convenient way to have sass in a project without having to build a build script.

Btw, using "import" in sass is of course possible. However, you should not use "import" to have at the end one big file instead of several small as the build script is responsible for merging those files together. Jarves does this optionally for you using the [asset compression](http://jarves.io/documentation/cook-book/create-a-theme#Load_additional_assets). For libraries like Twitter Bootstrap a index sass file that references all other make sense, or using imports to share configuration variables - which is still possible.

I'm going to merge this as everything is correct, I guess only the missing `web` folder is necessary. Thanks!

**Pull request merged**